hm-post-repeat.php

```diff
@@ -117,7 +120,14 @@ function publish_box_ui() {
 function post_states( $post_states, $post ) {
 
 	if ( is_repeating_post( $post->ID ) ) {
-		$post_states['hm-post-repeat'] = __( 'Repeating', 'hm-post-repeat' );
+
+		// If the schedule has been removed since publishing, let the user know.
+		if ( get_repeating_schedule( $post->ID ) ) {
+			$post_states['hm-post-repeat'] = __( 'Repeating', 'hm-post-repeat' );
+		} else {
+			$post_states['hm-post-repeat'] = __( 'Invalid Repeating Schedule', 'hm-post-repeat' );
```

Comment:
I added it to the post state and not the metabox, because one would have to click into the post to see the metabox. When multiple posts are repeating, this can be a bit annoying.
You're right that it's quite edge case, which is exactly why I added it to the post state, as it's non-obtrusive there, and will rarely pop up anyway.